### ISSUE TITLE
Exa fails with -l argument

### ISSUE THREAD

**Issue by Indy2222** [ogham/exa#687]
Exa panicked when called `exa -l` but it doesn't fail when I call just `exa`. I am running it on Arch Linux 

The output:

```
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: SystemTimeError(3600s)', src/libcore/result.rs:997:5
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
[1]    10786 abort (core dumped)  exa -lgumh --group-directories-first --color-scale
```

Exa version (installed from Arch repositories: `community/exa 0.9.0-1`):

```
$ exa --version
exa v0.9.0
```

My system:

```
$ uname -a
Linux indy 5.7.2-arch1-1 #1 SMP PREEMPT Wed, 10 Jun 2020 20:36:24 +0000 x86_64 GNU/Linux
```

Contents of the directory:

```
$ ls -la
total 12
drwxr-xr-x 3 indy indy 4096 Jun 18 10:50 .
drwxr-xr-x 6 indy indy 4096 Jun 18 10:50 ..
drwxr-xr-x 3 indy indy 4096 Jan  1  1970 CloudPipeline
```

And not so useful stack trace:

```
stack backtrace:
   0: <unknown>
   1: <unknown>
   2: <unknown>
   3: <unknown>
   4: <unknown>
   5: <unknown>
   6: <unknown>
   7: <unknown>
   8: <unknown>
   9: <unknown>
  10: <unknown>
  11: <unknown>
  12: <unknown>
  13: start_thread
  14: __GI___clone
```

**Comment by Freaky** [comment]
Should be fixed by #670.

Duplicate of #658, #596, #592, #567, #566, #556, #550, and possibly #428.

**Comment by ogham** [comment]
The fix for this has been merged and will go out with the next release.
